### DISCUSSION TITLE
Chocolatey packager doesn't show up in JReleaser's log output

### DISCUSSION THREAD

**Discussion by helpermethod** [jreleaser/jreleaser#350] [Q&A] [answered]
Hi @aalmiray,

I'm trying to release `connect-offset-reset` via Chocolatey, but I don't see anything happening at all.

Is there anything missing from the config?

https://github.com/helpermethod/connect-offset-reset/blob/main/jreleaser.yml

**Comment by aalmiray** [comment, marked as answer]
Chocolatey support for local builds is not done at the moment as it requires running on a windows environment. The current expectation is to push the generated manifest to a git repository and trigger a GH workflow that runs the `choco` command, like jbang does it at the moment https://github.com/jbangdev/jbang/blob/3f6688eaf0acef33e7198af77a61bfffe6a5c82b/.github/workflows/publish-packages.yml#L76-L107
